src/PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -465,15 +468,16 @@ private void Init()
             try
             {
                 // If already initialized, need not be on the UI thread
+                // Note that _initialized is only set to true once per VS instance
                 if (!_initialized)
                 {
-                    _initialized = true;
```

Comment:
I'm not following this change. The original code said that if the solution is not opened we are initialized because there is nothing there, and we will get the solution open event.

What is this change about?